lib/worker.coffee

```diff
@@ -44,3 +51,25 @@ module.exports = (text, attributes, options) ->
       stack: stack
 
   callback()
+
+registerBlocksPositions = (doc) ->
+  # Use Ruby API:
+  # because `doc.findBy()`` doesn't yet accept a filter function as a parameter.
+  # https://github.com/asciidoctor/asciidoctor.js/issues/282
+  blocks = Opal.block_send doc, 'find_by', (b) -> b.$lineno() isnt Opal.nil
```

Comment:
I love the "isnt". CoffeeScript is so cool. Many of the same reasons I love Ruby, but Ruby doesn't even have that.